fast64_internal/f3d/f3d_material.py

```diff
@@ -2440,6 +2442,61 @@ def reloadDefaultF3DPresets():
             update_preset_manual_v4(material, presetNameToFilename[material.f3d_mat.presetName])
 
 
+class UpdateColorSpacePopup(Operator):
+    bl_label = "Update Color Space"
+    bl_idname = "dialog.update_colorspace"
+    bl_description = "Update color space"
+    bl_options = {"UNDO"}
+
+    already_invoked = False  # HACK: used to prevent multiple dialogs from popping up
+
+    def invoke(self, context: Context, event: Event):
+        if UpdateColorSpacePopup.already_invoked:
+            return {"FINISHED"}
+        scene = context.scene
+        view_settings = scene.view_settings
+        # Check if color space is correct
+        if not scene.dont_ask_colorspace and (
+            scene.display_settings.display_device != "sRGB"
```

Comment:
I would check this before invoking the operator rather than on invoke? Seems more logical and better for performance